ansible/roles/test/files/ptftests/sad_path.py

```diff
@@ -431,7 +431,11 @@ def verify_bgp_dut_state(self, state='Idle'):
                 if key not in ['v4', 'v6']:
                     continue
                 self.log.append('Verifying if the DUT side BGP peer %s is %s' % (self.neigh_bgps[vm][key], states))
-                stdout, stderr, return_code = self.dut_connection.execCommand('show ip bgp neighbors %s' % self.neigh_bgps[vm][key])
+                if key == 'v4':
```

Comment:
Did you see any failures without this change? We use to have v4 and v6 neighbors included in the 'show ip bgp neighbors' output.

Reply:
We did experience issues in running tests against 201911 image. This may be a new change in the image.
However, for 202012 images, `show ip bgp neighbors` would include both ipv6 and ipv4 entries.

In latest 201911:
```
$ show ip bgp neighbors fc00::2
Usage: show ip bgp neighbors [OPTIONS] [IPADDRESS] [INFO_TYPE]

Error: fc00::2 is not valid ipv4 address
$
```

In latest 202012:
```
$ show ip bgp neighbors fc00::7e  
BGP neighbor is fc00::7e, remote AS 64600, local AS 65100, external link
 Description: ARISTA04T1
 Member of peer-group TIER1_V6 for session parameters
.....
```

Reply:
Finally, it turns out that running separate commands for ipv4 and ipv6 neighbors work on both version of the images.

Reply:
Interesting, we need to flag this behavior change across versions and see if something needs to be fixed in either version